### ISSUE TITLE
Stucked in Infinite search

### ISSUE THREAD

**Issue by JCik68** [steam3d/MagicPods-Windows#302]
Hello,

I am stucked in infinite search. I tried all solutions on your webpage but no luck - https://help.magicpods.app/issue-infinite-search/#solution-1

Some additional info - Bluetooth drivers 22.190.0.2 and AirPods 3.
If I unpair Airpods 3 and pair them again, MagicPods will find them. But if I disconnect them and connect them again afterwards, I am stucked in infinite search. Connection with PC is stable though and everything is OK.

Thanks for help

[MagicPodsServiceLog.txt](https://github.com/steam3d/MagicPods-Windows/files/10679265/MagicPodsServiceLog.txt)
[MagicPodsUILog.txt](https://github.com/steam3d/MagicPods-Windows/files/10679266/MagicPodsUILog.txt)

**Comment by steam3d** [comment]
Hi. Unfortunately, there is no way to do something from my side. When you connect AirPods, the service that receives information from AirPods stops outside the app.

I can't figure out who stop the service, I have only operation status information.

If you connect AirPods and put them in the case, does a popup window appear?

**Comment by JCik68** [comment]
Thanks for info, at least I know the problem is on my side. MagicPods was working OK on my old laptop. 

The one thing I am aware of that changed is that I have enabled Smart App Control on Windows 11 but not sure if this should be cause. I will try to reinstall Bluetooth driver again.

And popup window appears only the first time when I paired them. After I disconnected them and connected them again, popup window won’t appear and I am stuck in infinite search
